ext/zend_test/test.stub.php

```diff
@@ -72,6 +72,7 @@ enum ZendTestStringEnum: string {
         case Foo = "Test1";
         case Bar = "Test2";
         case Baz = "Test2\\a";
+        case FourtyTwo = "42";
```

Comment:
```suggestion
        case FortyTwo = "42";
```

"Forty" doesn't have a "u" in it, not even in Commonwealth English. 😉